### ISSUE TITLE
Add prettier(code formatter) to the project

### ISSUE THREAD

**Issue by makkoncept** [creativecommons/ccsearch-browser-extension#190]
This project has refrained from adding and forcing a particular code formatter on the developers. But, I think it's time to change this. The main reason behind this being a significant amount of inconsistent code  (as far as the formatting is concerned ) has made its way into the codebase. Most of the time even simple lint errors, which can be removed by proper formatting, are pushed to the PR and we become aware of those only after the Github Actions give lint warnings. Some formatting differences are not picked up by the linter and therefore adding a code formatter seems necessary.

TL;DR: To make the developers not worry about the formatting and to make sure that the code formatting is consistent in the code.

**Comment by techievivek** [comment]
A must for this project, will give a sense of relief for sure.
